tests/metrics/network/README.md

```diff
@@ -0,0 +1,74 @@
+# Networking tests for the Intel® Clear Containers runtime
+
+## Introduction
+
+Currently for the Clear Containers project, we have a series of network performance tests 
+that can be used as a basic reference in order to measure network essentials like:
+
+- Bandwidth
+- Latency
+- Jitter
+- Throughput
+- CPU % and memory consumption
+
+## Performance tools
+
+- Iperf/Iperf3 is a simple tool to let you measure the bandwidth and the quality of a network 
+link.
+
+- Nuttcp is a network performance tool that helps to determine the raw UDP network layer 
+throughput.
+
+- Smem is a memory reporting tool thats gives a user diverse reports on memory usage.
+
+## Networking tests
+
+- network-metrics
+
+Measures bidirectional network bandwidth using iperf. Bidirectional tests are used when 
+we want to test both servers for the maximum amount of throughput.
+
+- network-latency
+
+Measures latency using ping. The ping utility can be used as a tool to measure how long 
+it takes one packet to get from one point to another.
+
+- network-metrics-iperf3
```

Comment:
Could you explain how this test differs from the `network-metrics` test which also measures bandwidth using `iperf` (not `iperf3`)?